### ISSUE TITLE
Bulk insert does not throw error when one of the values fails a database condition

### ISSUE THREAD

**Issue by pellejacobs** [TimelordUK/node-sqlserver-v8#126]
When I run a bulk insert statement which returns the output, and the insert list includes values that are breaking database constraints, the query does not actually throw an error. 

A quick and dirty reproduction of the issue:

```js
const mssql = require('msnodesqlv8')
const connectionString = '...'
const runQuery = query => {
  return new Promise((resolve, reject) => {
    mssql.query(connectionString, query, (err, rows) => {
      if (err) reject(err)
      resolve()
    })
  })
}

const assert = require('assert')

;(async () => {
  await runQuery(`CREATE TABLE Persons (Name varchar(255) NOT NULL);`)

  // normal insert, runs fine
  await runQuery(`INSERT INTO [Persons] ([Name]) OUTPUT INSERTED.* VALUES (N'John')`)

  // failing insert, throws proper error
  let errorMessage
  try {
    await runQuery(`INSERT INTO [Persons] ([Name]) OUTPUT INSERTED.* VALUES (null)`)
  } catch (error) {
    errorMessage = error.message
  }
  assert(errorMessage.includes('Cannot insert the value NULL into column'))

  // bulk insert, throws proper error
  errorMessage = undefined
  try {
    await runQuery(`INSERT INTO [Persons] ([Name]) OUTPUT INSERTED.* VALUES (null), (N'John')`)
  } catch (error) {
    errorMessage = error.message
  }
  assert(errorMessage.includes('Cannot insert the value NULL into column'))

  // Problematic statement:
  // bulk insert with proper element first, does NOT throw an error
  errorMessage = undefined
  try {
    await runQuery(`INSERT INTO [Persons] ([Name]) OUTPUT INSERTED.* VALUES (N'John')', (null)`)
  } catch (error) {
    errorMessage = error.message
  }
  assert(errorMessage.includes('Cannot insert the value NULL into column'), 'Bulk insert should throw an error')

  // same bulk insert but without returning values, throws proper error again
  errorMessage = undefined
  try {
    await runQuery(`INSERT INTO [Persons] ([Name]) VALUES (N'John'), (null)`)
  } catch (error) {
    errorMessage = error.message
  }
  assert(errorMessage.includes('Cannot insert the value NULL into column'))

  await runQuery(`DROP TABLE Persons`)
})()
```

msnodesqlv8 version: 0.8.4
node version: 10.15.2

**Comment by TimelordUK** [comment]
hello

I added this test to bulk.js and ran on SQL Server 2014/2016 Node 10-13 x64/x86 with version 0.8.5

It seems to pass - can you try on latest version, I am not sure why I cannot reproduce it

```javascript
 test('bulk insert condition failure', testDone => {
    const createTableSql = 'CREATE TABLE Persons (Name varchar(255) NOT NULL)'
    const runQuery = query => {
      return new Promise((resolve, reject) => {
        theConnection.query(query, (err, rows) => {
          if (err) reject(err)
          resolve(rows)
        })
      })
    }
    const fns = [

      asyncDone => {
        theConnection.query('DROP TABLE Persons', () => {
          asyncDone()
        })
      },
      asyncDone => {
        theConnection.query(createTableSql, e => {
          assert.ifError(e)
          asyncDone()
        })
      },
      // normal insert, runs fine
      asyncDone => {
        runQuery(`INSERT INTO [Persons] ([Name]) OUTPUT INSERTED.* VALUES (N'John')`).then(() => {
          asyncDone()
        }).catch((e) => {
          assert.ifError(e)
        })
      },
      // failing insert, throws proper error
      asyncDone => {
        runQuery(`INSERT INTO [Persons] ([Name]) OUTPUT INSERTED.* VALUES (null)`).then(() => {
          assert(false)
        }).catch((e) => {
          assert(e.message.includes('Cannot insert the value NULL into column'))
          asyncDone()
        })
      },
      // bulk insert, throws proper error
      asyncDone => {
        runQuery(`INSERT INTO [Persons] ([Name]) OUTPUT INSERTED.* VALUES (null), (N'John')`).then(() => {
          assert(false)
        }).catch((e) => {
          assert(e.message.includes('Cannot insert the value NULL into column'))
          asyncDone()
        })
      },
      // Problematic statement:
      // bulk insert with proper element first, does NOT throw an error
      asyncDone => {
        runQuery(`INSERT INTO [Persons] ([Name]) OUTPUT INSERTED.* VALUES (null), (N'John')`).then(() => {
          assert(false)
        }).catch((e) => {
          assert(e.message.includes('Cannot insert the value NULL into column'), 'Bulk insert should throw an error')
          asyncDone()
        })
      },
      asyncDone => {
        runQuery(`INSERT INTO [Persons] ([Name]) VALUES (N'John'), (null)`).then(() => {
          assert(false)
        }).catch((e) => {
          assert(e.message.includes('Cannot insert the value NULL into column'))
          asyncDone()
        })
      }
    ]

    async.series(fns, () => {
      testDone()
    })
  })
```

**Comment by TimelordUK** [comment]
my apologies I have not copied the test correctly, I see the error now ..... will take a look

**Comment by TimelordUK** [comment]
this should be fixed in 0.8.6

**Comment by TimelordUK** [comment]
I think this is now fixed.
